carbonmark-api/src/models/Purchase.model.ts

```diff
@@ -43,6 +43,9 @@ export const PurchaseModel = Type.Object({
       vintage: Type.String({
         examples: ["2008"],
       }),
+      serialization: Type.Optional(
+        Type.String({ examples: ["ICR-ISL-354-78040-14-R-0-2021"] })
+      ),
```

Comment:
What is serialisation BTW?

Reply:
It's an additional ID or their only id?

Reply:
It's their only reliable id

Reply:
ok as long as we're tracking it. Happy for it to be done in a seperate PR

Reply:
Sure, keeping this combined tracking here: https://github.com/KlimaDAO/klimadao/issues/2093. Once this is merged I'll break them out